### ISSUE TITLE
Make new release

### ISSUE THREAD

**Issue by andresilva** [hack-ink/subalfred#842]
The current released version doesn't build with latest rust. Would it be possible to make a new release?

**Comment by aurexav** [comment]
> The current released version doesn't build with latest rust. Would it be possible to make a new release?

Absolutely, and I plan to refactor this project to perform a V2 upgrade. A lot has changed since Polkadot is transitioning to JAM. Feel free to request any new features you may need.

**Comment by aurexav** [comment]
I'll make next-gen subalfred, PolkAgent, with the power of AI. See you there!

**Comment by aurexav** [comment]
https://github.com/hack-ink/subalfred/releases/tag/v0.9.4
